s2i/s2i.yaml

```diff
@@ -28,9 +24,9 @@ spec:
 
   steps:
   - name: generate
-    image: ${inputs.params.BUILDER_IMAGE}
-    workingdir: /workspace/git-source
-    command: ['s2i', 'build', '${inputs.params.PATH_CONTEXT}', '${inputs.params.S2I_IMAGE}', '--as-dockerfile', '/gen-source/Dockerfile.gen']
+    image: quay.io/openshift-pipeline/s2i
```

Comment:
So we don't want to allow the user to override the s2i image?

Reply:
Do you think we should do ? looking around catalog, sometimes we allow it, sometimes we don't, so I wasn't sure :joy_cat:

Reply:
If I was an advanced user using this I would probably would override it with my own image that would do fancy stuff, but i guess we are not to that point yet,